### PR TITLE
make `GET /shutdown` API available during bootstrap

### PR DESCRIPTION
**Pull request by eugene-babichenko**

### CODE REVIEW

**Comment by mzabaluev** [comment]
Should it properly be `POST /shutdown`? Shutting the service down is not an idempotent request :nerd_face:

**Comment by eugene-babichenko** [comment]
@mzabaluev I do not want to deal with breaking API changes :P

**Comment by NicolasDP** [comment]
we could add a redirection to the right one, deprecating end points is not something unknown in the REST API world

**Comment by eugene-babichenko** [comment]
Is there a way to force a client to use POST instead of GET with a redirect? There is 303 for forcing GET, but I could not find anything like that for POST.

**Comment by eugene-babichenko** [comment]
jcli contains a breaking change now, should I keep `shutdown get` for compatibility?

**Comment by mzabaluev** [comment]
> jcli contains a breaking change now, should I keep `shutdown get` for compatibility?

I would support that. You can put in an `X-Warning:` header in the response that nobody will ever see :trollface:

File: doc/api/v0.yaml [hunk 1]
```diff
@@ -1032,6 +1032,14 @@ paths:
 
   /api/v0/shutdown:
     get:
+      deprecated: true
+      description: Starts node shutdown procedure
+      tags:
+        - utils
+      responses:
+        '200':
+          description: Success
```
Comment (NicolasDP) [inline doc/api/v0.yaml:1041]: Seems like something is missing:

> Operation should have an `operationId`.

Comment (eugene-babichenko) [reply doc/api/v0.yaml:1042]: done

**Pull request merged**